core/model/modelform.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * @package     RedCORE.Plugin
```

Comment:
we will need to remove redCORE from the docblocks in the future pulls since is not anymore part of it.